### ISSUE TITLE
showing only images searched by user

### ISSUE THREAD

**Issue by QuantumPhilosophy** [QuantumPhilosophy/anonymous-pumpkin#15]
### User Stories
**Nice-To-Have:** Showing images by user, each user will only see what images they searched for

### Tasks
- [ ] User Auth setup in Firebase

## Issue Links
**Depends On:** #4 start a Firebase project and initialize in Javascript - Complete
